browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1379,10 +1379,28 @@ Are you sure you want to do this?
         New private tab
       </message>
       <message name="IDS_BRAVE_NEW_TAB_PRIVATE_HEADER" desc="Header shown when a user opens a private tab explaining private mode">
-        This is a Private Tab
+        Private tab
       </message>
-      <message name="IDS_NEW_TAB_PRIVATE_DESC" desc="Message shown when a user opens a private tab explaining private mode">
-        Even though sites you visit in private tabs are not saved locally, they do not make you anonymous or invisible to your ISP, your employer, or to the sites you are visiting.
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_NO_TRACE" desc="No trace title shown when a user opens a private tab explaining private mode">
+        Leave no trace on your device
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_NO_TRACE_DESCRIPTION" desc="No trace description shown when a user opens a private tab explaining private mode">
+        Sites you visit in private windows won't appear in your history. Cookies, form data, and site data vanish once all private windows are closed.
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_VISIBILITY" desc="Browsing visibility title shown when a user opens a private tab explaining private mode">
+        Browsing visibility
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_VISIBILITY_DESCRIPTION" desc="Browsing visibility description shown when a user opens a private tab explaining private mode">
+        Note that -even in private windows- your browsing activity may still be visible to your ISP, network observers, or your employer (if you're using a work device/network).
```

Comment:
is it supposed to be like that? `-even in private windows-`. I can't open a link to figma design, not sure how do we have it there.

Reply:
Maybe because English is not my first language, but the use of hyphens feels weird to me as well.
Here is a screenshot from Figma:

![image](https://github.com/user-attachments/assets/0e0c0b39-4a41-479c-8c4d-997108290c8f)

Reply:
@aguscruiz could you please confirm that it's not a typo?

Reply:
Yeah, it's not a typo, we had that copy from @rmcfadden3 here:
https://www.figma.com/design/jt7tV0UsU7XhJXR2B8807E?node-id=37-7636#805038527